cmd/uplink/cmd/cat.go

```diff
@@ -38,9 +38,14 @@ func catMain(cmd *cobra.Command, args []string) (err error) {
 		return err
 	}
 
-	if u0.Host == "" {
+	if u0.Bucket() == "" {
 		return fmt.Errorf("No bucket specified. Please use format sj://bucket/")
 	}
 
-	return download(ctx, bs, u0, "-")
+	clp, err := fpath.New("-")
```

Comment:
Some better name like: `out`, `output`, `dst`

Reply:
Changed it everywhere in all commands.